lib/scala/interpreter-dsl/src/main/java/org/enso/interpreter/dsl/MethodProcessor.java

```diff
@@ -15,11 +14,12 @@
 import java.io.PrintWriter;
 import java.util.*;
 import java.util.stream.Collectors;
+import org.openide.util.lookup.ServiceProvider;
 
 /** The processor used to generate code from the {@link BuiltinMethod} annotation. */
 @SupportedAnnotationTypes("org.enso.interpreter.dsl.BuiltinMethod")
 @SupportedSourceVersion(SourceVersion.RELEASE_11)
-@AutoService(Processor.class)
+@ServiceProvider(service = Processor.class)
```

Comment:
Always happy to see my API being used. I have no explanation why it works and the other not, however.

Reply:
Yeah, I figured I will give it a try and it worked. I suppose it has something to do with some jdk incompatibility and autoservice failing to construct but I would have add some debugging to figure it out. Tried to repro with maven plugin but without success.